README.md

```diff
@@ -25,7 +25,14 @@ steps.forEach(step => {
 
 To solve an equation:
 ```js
-const steps = mathsteps.solveEquation('2x + 2x + x + x');
+const steps = mathsteps.solveEquation('2x + 3x = 35');
+
+steps.forEach(step => {
+    console.log("before change: " + step.oldEquation.print());  // e.g. before change: 2x + 3x = 35
```

Comment:
Uh, it's more readable on the same line. But then, later, if we keep doing this, we might end up bringing scroll bars in the code block, and people have to scroll horizontally to see the end of the comment.